easybuild/easyconfigs/o/oceanspy/oceanspy-0.2.0-foss-2022a.eb

```diff
@@ -0,0 +1,40 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'oceanspy'
+version = '0.2.0'
+
+homepage = 'https://github.com/hainegroup/oceanspy'
+description = """OceanSpy - A Python package to facilitate ocean model data analysis 
+and visualization.""" 
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('dask', '2022.10.2'),
```

Comment:
There is already `scikit-image-0.19.3-foss-2022a.eb` depending on dask v2022.10.0 in this toolchain. Please downgrade this dependency to that version.

Reply:
Done. I downgraded `dask` to the already existing `2022.10.0` version.